requirements.txt

```diff
@@ -10,6 +10,13 @@ Cartopy>=0.22.0
 pyproj>=3.4.1
 tueplots>=0.0.8
 plotly>=5.15.0
+torch-geometric>=2.5.2
```

Comment:
Will this play well with all the `pytorch-geometric` depdendencies (see point 5 in install instructions in the readme). If so I think it makes sense to include the `pytorch-geometric` dependency here, but we should also adjust the install instructions in the README.

Reply:
I'm not sure to be honest. In #37 am planning to implement install and testing on both cpu-only and gpu setups. So we can maybe postpone making a decision on where `pytorch-geometric` sits until then?

If so I would suggest you remove this from `requirements.txt` @SimonKamuk and instead put the installation into the github action following the steps that @joeloskarsson has put in the README. Sound ok?

Reply:
Note that the `torch_geometric` package itself does not need to be installed differently depending on if you are on CPU or GPU (see https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html). That is only the case for its optional dependencies these days. So I do not see a problem with just having it listed in `requirements.txt`. 

We should probably go over those optional dependencies as well, since I am not sure if they all are needed. Maybe something to do for #37 as well?

Reply:
Just to double check you're agreeing @joeloskarsson with my suggestion that the `torch-geometric` should be removed from `requirements.txt`, right? Just so @SimonKamuk knows what to do :)

Reply:
Yes, I think that is the easiest for now, and handle any restructuring of these dependencies in #37